proto/zepben/protobuf/hc/Syf.proto

```diff
@@ -94,4 +94,46 @@ message Syf {
      */
     string resultsProcessorConfig = 15;
 
+    /*
+     * Map of EnergyMeter IDs to their measurement zone info
+     */
+    map<string, MeasurementZoneInfo> mzInfo = 16;
+
+}
+
+enum MeasurementZoneType {
```

Comment:
To avoid the MZT_, this should be moved to a sub-package. maybe hc/measurement/?
You could put this and the MeasurementZoneInfo in that subpackage.

Reply:
Hi @kgreav I've fixed this.